modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm

```diff
@@ -0,0 +1,8 @@
+//////////////////////////////////////
+//lumossnowflakehair hair hair hair //
+//////////////////////////////////////
```

Comment:
come on bruh I told you to change this to something that isn't this ;_;

Reply:
no

Reply:
yes please make the box less memey